packages/suite/src/components/suite/modals/AddAccount/components/UnavailableInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { P } from '@trezor/components';
+import { Translation } from '@suite-components';
+import { Account } from '@wallet-types';
+import { TrezorDevice } from '@suite-types';
+import { isDesktop } from '@suite-utils/env';
+import { isDevEnv } from '@suite-common/suite-utils';
+
+const Wrapper = styled(P).attrs(() => ({
+    size: 'small',
+    textAlign: 'left',
+}))`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    margin: 20px 0;
+`;
+
+interface UnavailableInfoProps {
+    device: TrezorDevice;
+    accountType: Account['accountType'];
+}
+
+const UnavailableMessage = ({ device, accountType }: UnavailableInfoProps) => {
```

Comment:
This could just be a function returning the `id` or a props object passed to `Translation` rather than a full component. It would seem more readable to me.